core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingDataService.js

```diff
@@ -0,0 +1,200 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the graphical state editor.
+ */
```

Comment:
Please add a newline after this.

Reply:
@shubha1593  I have added the new line .

Reply:
Thanks @ishucr7. Also can you update this fileoverview comment, better move the comment below, here. For example it would be `@fileoverview Service for training data that, given an .....`. 
(Sorry didn't catch it before!)